bib/internal/podmanutils/podmanutils.go

```diff
@@ -0,0 +1,38 @@
+package podmanutils
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+)
+
+// envPath is written by podman
+const envPath = "/run/.containerenv"
+
+// rootlessKey is set when we are rootless
+const rootlessKey = "rootless=1"
+
+// IsRootless detects if we are running rootless in podman;
+// other situations (e.g. docker) will successfuly return false.
+func IsRootless() (bool, error) {
+	buf, err := os.ReadFile(envPath)
+	if err != nil {
```

Comment:
(niptick^3) we could maybe avoid the nested if and write this as
```go
	buf, err := os.ReadFile(envPath)
	if errors.Is(err, fs.ErrNotExist) {
		return false, nil
	}
	if err != nil {
		return false, err
	}
...
```
but maybe I'm totally overthinking it :)

Reply:
I've usually nested them...not sure if there's a truly standard style here but glancing at e.g. podman including some vendors I mostly see nesting for this.

Reply:
Yeah, totally fine, sorry for being too nitpicky